### DISCUSSION TITLE
Questions to be graphed for a paper

### DISCUSSION THREAD

**Discussion by bethac07** [GloBIAS-BioimageAnalysts/Survey_2024#13] [Ideas]
Will make one per comment below, to keep discussion nicely threaded. Can link to issue(s) in each one.

**Comment by bethac07** [comment]
Career stage

**Comment by bethac07** [comment]
What time spent on

**Reply by bethac07** [reply]
See also #11

**Comment by ChristaGW** [comment]
Using Q2 (which do you work in) - filter for anyone working in a facility - check if answered Q&/Q7 (should only be answered by people in facilities) - using Q6 (or Q7) - and overlay with responses for Q1: Can we get an idea how much BioImage Analysis people in Facilities do compared with other aspects of their work?

**Comment by ChristaGW** [comment]
Q2 (where do you work) versus Q3 (where do IA sit) - is there a trend depending of university/ research lab/ profit/ not for profit (would require summing categories of Q2). Maybe also cross check results versus demographic question: which sector do you work in.

**Comment by ChristaGW** [comment]
Q4 operating model of your facility - how much time do people have for own projects (Q7)
